src/utils/getUserById.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/user';
+
+export function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) as User;
```

Comment:
Don't use " as ", it's bad practice. User can be "null" and it's ok

Reply:
Good point. Find method can possibly return undefined